ci-operator/step-registry/stackrox/collector/update-support-packages/setup/stackrox-collector-update-support-packages-setup-ref.yaml

```diff
@@ -0,0 +1,17 @@
+ref:
+  as: stackrox-collector-update-support-packages-setup
+  commands: stackrox-collector-update-support-packages-setup-commands.sh
+  from: src
+  credentials:
+    - mount_path: /tmp/secret/stackrox-collector-e2e-tests
+      name: stackrox-collector-e2e-tests
+      namespace: test-credentials
+  resources:
+    requests:
+      cpu: 2000m
```

Comment:
Are resource requests needed here? Do we need 2 cores and 4GB of memory to launch the VM?